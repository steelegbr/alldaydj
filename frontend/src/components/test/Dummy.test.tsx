import React from 'react';
import { mount } from 'enzyme';
import Dummy from 'components/test/Dummy';

describe('dummy component', () => {
  function mountComponent() {
    return mount(<Dummy />);
  }

  it('snapshot', () => {
    const dummy = mountComponent();
    expect(dummy).toMatchSnapshot();
  });
});
