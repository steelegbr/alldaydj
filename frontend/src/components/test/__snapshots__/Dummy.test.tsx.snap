// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dummy component snapshot 1`] = `
<Dummy>
  <ForwardRef(Typography)
    variant="h1"
  >
    <MuiTypographyRoot
      as="h1"
      className="MuiTypography-root MuiTypography-h1"
      ownerState={
        Object {
          "align": "inherit",
          "children": "Hello from AllDay DJ",
          "className": undefined,
          "color": undefined,
          "component": undefined,
          "gutterBottom": false,
          "noWrap": false,
          "paragraph": false,
          "sx": Object {
            "color": undefined,
          },
          "variant": "h1",
          "variantMapping": Object {
            "body1": "p",
            "body2": "p",
            "h1": "h1",
            "h2": "h2",
            "h3": "h3",
            "h4": "h4",
            "h5": "h5",
            "h6": "h6",
            "inherit": "p",
            "subtitle1": "h6",
            "subtitle2": "h6",
          },
        }
      }
      sx={
        Object {
          "color": undefined,
        }
      }
    >
      <h1
        className="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
      >
        Hello from AllDay DJ
      </h1>
    </MuiTypographyRoot>
  </ForwardRef(Typography)>
  <ForwardRef(Typography)
    paragraph={true}
  >
    <MuiTypographyRoot
      as="p"
      className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
      ownerState={
        Object {
          "align": "inherit",
          "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed egestas eros arcu, eu placerat arcu lacinia ut. Sed laoreet imperdiet risus, quis vulputate sapien maximus ut. In facilisis vitae sapien eu sagittis. Donec volutpat massa eget mi accumsan, sit amet placerat turpis sagittis. Donec egestas arcu vel euismod eleifend. Quisque et nulla quis nisi ullamcorper bibendum quis nec eros. Cras tincidunt justo quis rhoncus luctus. Vivamus et eleifend lacus. Sed malesuada consequat mauris, ac pretium odio. Vestibulum sed ipsum eget nisi vehicula vehicula vitae sed erat. Fusce imperdiet quam quis orci pretium, at dignissim enim posuere. Nunc sed ex volutpat, suscipit ipsum vel, viverra est. Vivamus massa erat, tempus a eleifend nec, placerat et dui. Suspendisse auctor velit nulla, quis consectetur lorem interdum eget.",
          "className": undefined,
          "color": undefined,
          "component": undefined,
          "gutterBottom": false,
          "noWrap": false,
          "paragraph": true,
          "sx": Object {
            "color": undefined,
          },
          "variant": "body1",
          "variantMapping": Object {
            "body1": "p",
            "body2": "p",
            "h1": "h1",
            "h2": "h2",
            "h3": "h3",
            "h4": "h4",
            "h5": "h5",
            "h6": "h6",
            "inherit": "p",
            "subtitle1": "h6",
            "subtitle2": "h6",
          },
        }
      }
      sx={
        Object {
          "color": undefined,
        }
      }
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph css-z2eky3-MuiTypography-root"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed egestas eros arcu, eu placerat arcu lacinia ut. Sed laoreet imperdiet risus, quis vulputate sapien maximus ut. In facilisis vitae sapien eu sagittis. Donec volutpat massa eget mi accumsan, sit amet placerat turpis sagittis. Donec egestas arcu vel euismod eleifend. Quisque et nulla quis nisi ullamcorper bibendum quis nec eros. Cras tincidunt justo quis rhoncus luctus. Vivamus et eleifend lacus. Sed malesuada consequat mauris, ac pretium odio. Vestibulum sed ipsum eget nisi vehicula vehicula vitae sed erat. Fusce imperdiet quam quis orci pretium, at dignissim enim posuere. Nunc sed ex volutpat, suscipit ipsum vel, viverra est. Vivamus massa erat, tempus a eleifend nec, placerat et dui. Suspendisse auctor velit nulla, quis consectetur lorem interdum eget.
      </p>
    </MuiTypographyRoot>
  </ForwardRef(Typography)>
</Dummy>
`;
